### DISCUSSION TITLE
Android Auto

### DISCUSSION THREAD

**Discussion by ryandikdan** [UnicornsOnLSD/finamp#530] [General]
Thanks for this great Jellyfin music app! (Which doesn't stop playing music when the screen goes off, like the normal Jellyfin app haha). I was wondering how much work it would be to add Android Auto functionality. The Jellyfin app actually has this, but it's broken, so it only shows a random smattering of artists in my collection. It also doesn't sync between the android auto app and the app on my phone. 

Let me know if you all think this is something doable or if it's a ton of work. I've been looking into it a bit myself, but I'm not much of an app developer.

**Comment by UnicornsOnLSD** [comment]
Android Auto discussion is at #24, it's a pretty hard thing to do but someone's made a proof of concept that looks promising

**Reply by ryandikdan** [reply]
Gotcha, you guys are crazy! (Impressive) Sorry I didn't check the issues, but thanks for pointing me in the right direction. I'll test out what's there!
